source/robot_model/include/robot_model/Model.hpp

```diff
@@ -220,7 +235,7 @@ inline void Model::init_model() {
   // get the joint names
   this->joint_names_ = this->robot_model_.names;
   // the first element is not a joint so remove it
-  this->joint_names_.erase(this->joint_names_.begin());
+//  this->joint_names_.erase(this->joint_names_.begin()); // dont erase this because the universe joint is everywhere
```

Comment:
I know the 'universe' joint is useless and may lead to confusion but I disagree to remove it from the joint names and not from everywhere else because its still there in the robot_model_.

Reply:
Seems reasonable to me. In agreement with your earlier comment, I don't think we need to have a private `joint_names_` member if we already have access to `robot_model_.names`. Remove this line makes the two completely equivalent.

Reply:
Maybe, we could introduce some public members though...https://github.com/epfl-lasa/control_libraries/issues/13